instrumentation/apache-camel-2.20/javaagent/src/test/groovy/io/opentelemetry/javaagent/instrumentation/apachecamel/aws/SnsCamelTest.groovy

```diff
@@ -24,7 +24,11 @@ class SnsCamelTest extends AgentInstrumentationSpecification {
     String queueName = "snsCamelTest"
     def camelApp = new CamelSpringApp(awsConnector, SnsConfig, ImmutableMap.of("topicName", topicName, "queueName", queueName))
 
-    def (queueUrl, topicArn) = setupTestInfrastructure(queueName, topicName)
+    // TODO: def (queueUrl, topicArn) fails to compile, switch back when this is fixed in spock
```

Comment:
It is already fixed, just not released yet: https://github.com/spockframework/spock/pull/1333

Probably makes sense to link that PR/issue here.